django/db/backends/base/features.py

```diff
@@ -227,6 +227,12 @@ class BaseDatabaseFeatures(object):
     # Place FOR UPDATE right after FROM clause. Used on MSSQL.
     for_update_after_from = False
 
+    # Combinatorial flags
+    supports_select_union = True
```

Comment:
How did you choose the defaults? I think it would be simplest to default to True and have backends opt-out as needed (that eliminates the risk that a backend fails to opt-in) but feel free to explain your thinking.